client/source/js/modules/common/normalise-height-directive.js

```diff
@@ -0,0 +1,122 @@
+define(['angular', 'jquery', 'underscore'], function (angular, $, _) {
+  'use strict';
+
+  return angular.module('app.common.normalise-height', [])
+
+    /**
+     * This directive normalises the height to the tallest element having the
+     * `normalise-height` attribute currently available in the DOM.
+     *
+     * In this example both elements will end up with a height of 200px:
+     *  <div style="height: 120px" normalise-height></div>
+     *  <div style="height: 200px" normalise-height></div>
+     *
+     * Important: Any element that re-draws itself & changes the height
+     * during this process must trigger a jQuery `draw` event for the new
+     * height to be taken into account.
```

Comment:
`draw` is a good method name for an object but events with names that are informative have the form of a past action. 

That's because, as you probably guessed, they are trying to signal that an something (an event) happened in the past. It probably just happened 2 nano-seconds ago, but is still the past.